content/articles/redirect-heroku.markdown

```diff
@@ -77,9 +77,27 @@ In this case, [you cannot use our redirector service](/articles/redirector-https
 In order to redirect using HTTPS, you need to have a certificate that covers both the www and not-www (root domain) host names. You can purchase a single-name certificate for `www` or a wildcard certificates for `*` [to cover the root domain](/articles/ssl-certificate-host name/) with the same certificate.
 </warning>
 
-
 <div class="section-steps" markdown="1">
 ##### To handle the redirect at Heroku {#redirect-https-application}
+##### For Heroku SSL
+
+1.  Add both host names to the Heroku application:
+
+    - `$ heroku domains:add example.com`
+    - `$ heroku domains:add www.example.com`
+
+1. [Order the SSL certificate](https://devcenter.heroku.com/articles/ssl-certificate-dnsimple) and add it to Heroku(https://devcenter.heroku.com/articles/ssl):
+
+    - `$ heroku certs:add server.crt server.key`
+
+1.  Go to the record editor and two DNS records, one for each host name, **pointing to the Heroku SSL endpoint**:
+
+    - Add an [ALIAS record](/articles/alias-record) to point `example.com` to Heroku. Leave the _Name_ of the record empty and set the _Content_ field to the SSL endpoint `encrypted-application.herokudns.com`
```

Comment:
Actually, for herokudns the name always match the domain, not the app name. Hence:

    example.com.herokudns.com

or

    www.example.com.herokudns.com